docs/topics/mta-web-creating-jira-issues-for-migration-wave.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * docs/web-console-guide/master.adoc
+
+:_content-type: PROCEDURE
+[id="mta-web-creating-jira-issues-for-migration-wave_{context}"]
+= Creating Jira issues for a migration wave
+
+You can use a migration wave for creating Jira issues automatically for each application assigned to it.
+
+.Procedure
+
+. In *Migration* view, click *Migration waves*.
+. From the *Options* menu (three dots) of the migration wave for which you want to create Jira issues, select *Export to Issue Manager*.
++
```

Comment:
We usually write this as "Click the Options menu {kebab} to the right of ... " with Options not bolded (unless it is actually called "Options" in the UI.

{kebab} is the attribute for the three dots in the windup-documentation repo -- it renders as the three dots. 

Please use the attribute and consider the rewrite I suggested for the sake of consistency in the repo.